docs/GettingStarted.md

```diff
@@ -84,9 +85,10 @@ Available settings are:
 * ``auto_instrument_redis``: if set to ``true`` Redis calls will be traced as such. Calls to Redis cache may be
   still instrumented but you will not have the detail of low-level Redis calls.
 * ``default_service``: set the service name used when tracing application requests. Defaults to ``rails-app``
+* ``default_controller_service``: set the service name used when tracing a Rails action controller. Defaults to ``rails-controller``
```

Comment:
yes, that's part of our "breaking change", that basically doesn't break anything, but changes the previous behavior adding this layer in the middle